app/serializers/metadata_serializer.rb

```diff
@@ -4,7 +4,11 @@ def initialize(metadata)
   end
 
   def filters_to_json
```

Comment:
Ah sorry, I've just seen this :) Can then the method on the metadata model go away?
Also, can we please make this so that accepts the data as input? So this could work with different data